app/scripts/metamask-controller.js

```diff
@@ -4940,6 +4948,7 @@ export default class MetamaskController extends EventEmitter {
         ),
         getAccountType: this.getAccountType.bind(this),
         getDeviceModel: this.getDeviceModel.bind(this),
+        isConfirmationRedesignEnabledFn,
```

Comment:
Very minor, but we have plenty of `isX` callbacks so the `Fn` might not be needed.

Reply:
👍🏼  works with me [cde5b5e](https://github.com/MetaMask/metamask-extension/pull/24182/commits/cde5b5ee0ffcc9cdc1ae6a1bc155a1f41eec56cb)